syntax="proto2";
package ClioServerStatus;

message ServerStatus {
	required string proc_name = 1;
	required string host_name = 2;
	required string ip_address = 3;
	
	enum E_STATUS {
		UP = 0;
		DOWN = 1;
		SUSPEND = 2;
	}

	required E_STATUS status = 4;
}
